build_tools/circle/build_doc.sh

```diff
@@ -112,7 +112,10 @@ fi
 wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
    -O miniconda.sh
 chmod +x miniconda.sh && ./miniconda.sh -b -p $MINICONDA_PATH
-export PATH="$MINICONDA_PATH/bin:$PATH"
+export PATH="/usr/lib/ccache:$MINICONDA_PATH/bin:$PATH"
```

Comment:
Should `/usr/lib/ccache/bin` be in the path?

Reply:
That path seem distribution dependent, but for the docker images we use for docs and PyPy is `/usr/lib/ccache/` without the `/bin/`,
```bash
$ docker run --rm -it circleci/python:3.6 /bin/bash              
Unable to find image 'circleci/python:3.6' locally
3.6: Pulling from circleci/python
[...]
Digest: sha256:6949fdd331307a8cb36c1e3a39ef66c3ab3ec947a924638b1d4d57595bcf6942
Status: Downloaded newer image for circleci/python:3.6
circleci@a65d8c1da300:/$ sudo apt install ccache
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Suggested packages:
  distcc
The following NEW packages will be installed:
  ccache
0 upgraded, 1 newly installed, 0 to remove and 2 not upgraded.
Need to get 136 kB of archives.
After this operation, 311 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian stretch/main amd64 ccache amd64 3.3.4-1 [136 kB]
Fetched 136 kB in 0s (907 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package ccache.
(Reading database ... 33759 files and directories currently installed.)
Preparing to unpack .../ccache_3.3.4-1_amd64.deb ...
Unpacking ccache (3.3.4-1) ...
Setting up ccache (3.3.4-1) ...
Updating symlinks in /usr/lib/ccache ...
circleci@a65d8c1da300:/$ ls /usr/lib/ccache/
c++  c89-gcc  c99-gcc  cc  g++  g++-6  gcc  gcc-6  x86_64-linux-gnu-g++  x86_64-linux-gnu-g++-6  x86_64-linux-gnu-gcc  x86_64-linux-gnu-gcc-6
```